.github/workflows/release.yml

```diff
@@ -29,23 +43,24 @@ jobs:
         with:
           version: 10
 
-      - name: download mesh-thumbnail (ubuntu only)
-        if: matrix.platform == 'ubuntu-22.04'
+      - name: download mesh-thumbnail (not windows)
+        if: matrix.os != 'windows-latest'
         run: |
           mkdir external-binaries
           cd external-binaries
-          wget -O mesh-thumbnail-x86_64-unknown-linux-gnu https://github.com/suchmememanyskill/mesh-thumbnail/releases/latest/download/mesh-thumbnail
-          chmod a+x mesh-thumbnail-x86_64-unknown-linux-gnu
+          wget https://github.com/${{ env.MESH_THUMBNAIL_REPO }}/releases/latest/download/mesh-thumbnail-${{ matrix.target }}
+          chmod a+x mesh-thumbnail-${{ matrix.target }}
 
       - name: download mesh-thumbnail (windows only)
-        if: matrix.platform == 'windows-latest'
+        if: matrix.os == 'windows-latest'
         run: |
           mkdir external-binaries
           cd external-binaries
-          curl -L https://github.com/suchmememanyskill/mesh-thumbnail/releases/latest/download/mesh-thumbnail.exe -o mesh-thumbnail-x86_64-pc-windows-msvc.exe
+          #TODO: build mesh-thumbnail for windows with msvc instead of gnu in mesh-thumbnail
```

Comment:
Didn't you complete this TODO:

Reply:
not quite: https://github.com/suchmememanyskill/mesh-thumbnail/pull/3/files#diff-d0777657fa3fd81d23aaf7273e58aee453b04e67882517900c56daeef9b3e4c1R25

mesh-thumbnail still builds using the gnu toolchain in that PR. the matrix.target here is still `target: x86_64-unknown-linux-gnu`, but I download it as `mesh-thumbnail-x86_64-pc-windows-msvc.exe` because this repo uses a windows-latest runner to build mesh-organizer, and it chokes on the gnu/msvc filename mismatch.